### PR TITLE
add arguments for deployment metadata

### PR DESCRIPTION
**Pull request by meik99**

### CODE REVIEW

**Review by DTMad — DISMISSED** [review]

File: controllers/oneagent/daemonset_test.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
+package oneagent
+
+import (
+	"testing"
+
+	"github.com/Dynatrace/dynatrace-operator/api/v1alpha1"
```
Comment (lrgar) [inline controllers/oneagent/daemonset_test.go:6]:
```suggestion
	dynatracev1alpha1 "github.com/Dynatrace/dynatrace-operator/api/v1alpha1"
```

File: controllers/oneagent/deployment_metadata.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+import "fmt"
+
+const (
+	orchestrationTech = "Operator"
+	argumentPrefix    = `--set-deployment-metadata=`
+
+	keyOperatorScriptVersion = "script_version"
+	keyContainerImageVersion = "container_image_version"
+	keyOrchestratorId        = "orchestrator_id"
```
Comment (lrgar) [inline controllers/oneagent/deployment_metadata.go:11]:
```suggestion
	keyOrchestratorID        = "orchestrator_id"
```

File: controllers/oneagent/deployment_metadata.go [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+	keyOrchestrationTech     = "orchestration_tech"
+)
+
+type deploymentMetadata struct {
+	operatorScriptVersion string
+	orchestratorId        string
```
Comment (lrgar) [inline controllers/oneagent/deployment_metadata.go:17]:
```suggestion
	orchestratorID        string
```

**Review by lrgar — DISMISSED** [review]

File: controllers/oneagent/daemonset_test.go (hunk 1 above)
Comment (meik99) [reply controllers/oneagent/daemonset_test.go]: added import alias

File: controllers/oneagent/deployment_metadata.go (hunk 2 above)
Comment (meik99) [reply controllers/oneagent/deployment_metadata.go]: renamed

File: controllers/oneagent/deployment_metadata.go (hunk 3 above)
Comment (meik99) [reply controllers/oneagent/deployment_metadata.go]: renamed

**Review by lrgar — APPROVED** [review]

**Review by DTMad — APPROVED** [review]

**Pull request merged**